### ISSUE TITLE
Change build network API

### ISSUE THREAD

**Issue by drvinceknight** [CiwPython/Ciw#117]
As we have discussed

```
parameters = {"c": 2, "n": 3}

def build_network(c=1, n=1, schedule=None, **kwargs):
    """
    Vince knows a little bit about this stuff
    """
    return (c, n)

build_network(Arrival_distributions={'Class 0': [['Exponential', 3.0]]},
              Service_distributions={'Class 0': [['Exponential', 5.0]]},
              Transition_matrices={'Class 0': [[0.0]]},
              Number_of_servers=[1])

build_network(**parameters)
```

**Comment by geraintpalmer** [comment]
Addressed in #118

**Comment by geraintpalmer** [comment]
Implemented in 8ff343dd09ab745cff2d0ff4a1c4bf16c738655b
